help/marking-messages-as-read.md

```diff
@@ -94,6 +94,8 @@ stream or topic as read**.
 
 {end_tabs}
 
+## Bankruptcy
```

Comment:
I think you can drop this hunk and @drrosa can do the documentation part in a follow-up PR.

Reply:
Dropped!

Reply:
OK -- and I think we can just let @alya file a new issue for whatever we want to do with documentation here.

Reply:
@timabbott and I chatted, and we don't currently think there's anything documentation that's needed.